modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/UserNotificationSettings.tsx

```diff
@@ -229,7 +229,7 @@ export function UserNotificationSettings({
 							Liferay.Language.get(
 								'use-terms-to-configure-recipients-for-this-notification-x'
 							),
-							'[%ENTRY_CREATOR%]',
+							'[%CURRENT_USER_ID%][%OBJECT_AUTHOR_ID%]',
```

Comment:
Hey @Valmir-unicap (and also @p-albuquerque), The right placeholder is only `[%OBJECT_AUTHOR_ID%]`, not `[%CURRENT_USER_ID%][%OBJECT_AUTHOR_ID%]`. Please, fix it.